src/test/java/core/basesyntax/dao/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    private static final String TEST_FILE = "src/test/resources/fromFileTest.csv";
+    private final CsvFileReader csvFileReader = new CsvFileReaderImpl();
+
+    @Test
+    public void readTransactionsTestValidData_Ok() {
+        List<FruitTransaction> actual
+                = csvFileReader.readTransactions(TEST_FILE);
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
```

Comment:
What about `List<FruitTransaction> actual` content? Do not we need to check what is inside?